advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md

```diff
@@ -39,7 +39,7 @@ Conformance levels get mentioned at a couple of other points in these lessons, a
 
 ### (Before) implementing accessibility
 
-The lessons that follow dive into several concepts that can help you improve the accessibility of your websites, but they don't cover *every* facet of accessibility. The goal of these lessons is just to introduce you to some of the more common concepts that you should get into the habit of checking for moving forward.
+The lessons that follow dive into several concepts that can help you improve the accessibility of your websites, but they don't cover _every_ facet of accessibility. The goal of these lessons is just to introduce you to some of the more common concepts that you should get into the habit of checking for moving forward.
```

Comment:
For the markdown we use asterisk. Not the _ _. If you could revert it back to the asteriks that would be great! You can check it out in this part of the [style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#markdown-styling) if your curious.

Reply:
Understood! Prettier is VERY opinionated. Apologize for the oversight.

Reply:
> Understood! Prettier is VERY opinionated. Apologize for the oversight.

A small tip, prettierignore this repo (or even better, bind formatting to a keyboard shortcut and don't rely on format-on-save if you're using format-on-save) when working on stuff in this repo, as the style guide does not follow what Prettier would do.

Reply:
I like the idea of binding to a key instead of format-on-save. Will do 👍🏻